testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -124,7 +126,7 @@ def test_strings(self):
         assert_equal(util.iterable(u"unicode string"), False)
 
 
-class TestFilename(object):
+class TestFilename(TestCase):
```

Comment:
So we need this for a swift conversion and then remove it one by one again later. check